scripts/qa_crowbarsetup.sh

```diff
@@ -5735,6 +5741,25 @@ function onadmin_batch
 {
     pre_hook $FUNCNAME
 
+    if [[ $hacloud = 1 ]] ; then
+        onadmin_set_source_variables
+        cluster_node_assignment
+
+        if [[ $want_sbd = 1 ]] ; then
+            local cluster
+            local clustername
+            local nodealias
+            for clustername in data network services ; do
+                eval "cluster=\$clusternodes$clustername"
+                for node in $cluster ; do
+                    nodealias=$(get_node_alias $node)
+                    sbd_device=$(ssh $node echo '/dev/disk/by-id/scsi-$(lsscsi -i |grep LIO|head -n 1| tr -s " " |cut -d " " -f7)')
+                    sed -i "s|@@sbd_device_${clustername}_${nodealias}@@|${sbd_device}|g" ${scenario}
```

Comment:
We already have ##name## placeholders used below. Any reason not to follow the same syntax here?

Reply:
I borrowed this from [one of the QA jobs](https://ci.suse.de/view/Cloud/view/QA/job/cloud-mkphyscloud-qa-scenario-2a/configure), but I see your point. I also plan to update that job to use this feature instead of duplicating it in the job script.
I switched to using the `##name##` notation instead of `@@name@@`